offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonView.groovy

```diff
@@ -371,14 +374,23 @@ class CreatePersonView extends FormLayout implements Resettable{
         return titleCombobox
     }
 
+    /**
+     * Initializes the default values to be set in the components of the CreatePersonVieW
+     * @return
+     */
+    private void initDefaultValues() {
+        titleField.setSelectedItem(AcademicTitle.NONE.toString())
+        createPersonViewModel.academicTitle = AcademicTitle.NONE
```

Comment:
```suggestion
        createPersonViewModel.academicTitle = AcademicTitle.NONE
```
The first line is not needed if you assume the ViewModel being bound correctly.

Reply:
You're right 👍